app/views/templates/tests-by-category-widget.html.slim

```diff
@@ -0,0 +1,66 @@
+.panel.panel-default
+  .panel-heading
+    .row
+      .col-md-6
```

Comment:
This `.row` and the two `.col-md-6` are completely superfluous. Also, the button drops below the title when the screen is smaller.

Put the right-floating button before the `h3` (otherwise it's displayed below it) and add `.clearfix` to the `.panel-heading`.